README.md

```diff
@@ -124,6 +124,8 @@ jobs:
         repository: elementary
         flat-manager-url: https://flatpak-api.elementary.io
         token: some_very_hidden_token
+        end-of-life: "The application has been renamed to..."
+        end-of-life-rebase: "org.gnome.zbrown.Palette=org.gnome.zbrown.NewPalette"
```

Comment:
I think we should just put the new app-id here, why do we need the old app-id?

Reply:
I assume that `flat-manager-client` expects this format:

https://github.com/flatpak/flat-manager/blob/f2c1f719b35cb48b5d2b7282b66c8a7e166147b7/src/jobs.rs#L377

Reply:
If I look at the line longer

```rust
Some(format!("--end-of-life-rebase={}={}", app_ref.ref_name.split('/').nth(1).unwrap(), endoflife_rebase))
```

probably only `end_of_life_rebase` is read from the command line...